Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -13,14 +13,24 @@
 with open(broker_ca_bundle, "w") as text_file:
     text_file.write(demisto.params()['broker_ca_bundle'])
 
+if not demisto.params()['broker_ca_bundle'].startswith('-----BEGIN CERTIFICATE-----'):
```

Comment:
move to another function (duplicate codes)
check for entire format (begin, end, certificate structure)